### DISCUSSION TITLE
window.onload not getting triggered

### DISCUSSION THREAD

**Discussion by Niocas** [vikejs/vike#958] [Discussion]
I have the same code in my vite-react-ts project, now I tried to use the same component in my vite-plugin-ssr project,
but window.onload never gets triggered. It prints "Checkpoint 1", but "Checkpoint 2" is never reached. Is there any
solution for that?

```
useEffect(() => {
        console.log("Checkpoint 1");
        window.onload = () => {
            console.log("Checkpoint 2");
        };
    }, []);
```

**Comment by brillout** [comment]
It's more of a React question than a VPS one.
